controllers/controllers.js

```diff
@@ -15,3 +15,15 @@ exports.getArticles = (req, res, next) => {
         })
         .catch(next)
 }
+exports.getArticleById = (req, res, next) => {
+    selectArticleById(req)
+        .then((article) => {
+            if (article !== undefined) {

```

Comment:
This .then block should be only sending the response. The error handling should be dealt with in the model.